spk/sonarr/Makefile

```diff
@@ -0,0 +1,57 @@
+SPK_NAME = sonarr
```

Comment:
So if I get this right (I ain't a sonarr user), the v3 is actually called nzbdrone while v4 will be called sonarr?

Reply:
Yes, that is correct